### PR TITLE
adapt swe to the new hvdc + shifter lib

### PR DESCRIPTION
**Pull request by TheoPascoli**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/shift/ZonalScalableProvider.java [hunk 1]
```diff
@@ -17,35 +12,42 @@
...
             Scalable scalableFR = getCountryGeneratorsScalableForFR(network);
-            zonalScalable.addAll(new ZonalDataImpl<>(Collections.singletonMap(eicFR, scalableFR)));
+            zonalData.addAll(new ZonalDataImpl<>(Collections.singletonMap(eicFR, scalableFR)));
+        }
+        return zonalData;
+    }
+
+    public ZonalData<Scalable> importGlsk(String glskUrl, Network network, Instant instant) {
+        try (InputStream glskResultStream = new URL(glskUrl).openStream()) {
+            synchronized (LOCK_GLSK) {
```
Comment (vbochetRTE) [inline gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/shift/ZonalScalableProvider.java:44]: Why do we need to take care of concurrent access here?

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/NetworkShifterProvider.java [hunk 2]
```diff
@@ -24,30 +27,31 @@
  * @author Ameni Walha {@literal <ameni.walha at rte-france.com>}
  */
 @Service
+@Import(ProcessConfiguration.class)
```
Comment (vbochetRTE) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/NetworkShifterProvider.java:30]: Why do we need this annotation?

File: gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/CneFileExportService.java [hunk 3]
```diff
@@ -54,6 +57,7 @@
  */
 
 @Service
+@Import(ProcessConfiguration.class)
```
Comment (vbochetRTE) [inline gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/CneFileExportService.java:60]: Why do we need this annotation?

File: gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/shift/ScalableGeneratorConnector.java [hunk 4]
```diff
@@ -4,25 +4,16 @@
...
 import com.farao_community.farao.dichotomy.api.exceptions.ShiftingException;
 import com.powsybl.glsk.commons.ZonalData;
 import com.powsybl.iidm.modification.scalable.Scalable;
-import com.powsybl.iidm.network.Bus;
-import com.powsybl.iidm.network.Country;
-import com.powsybl.iidm.network.Generator;
-import com.powsybl.iidm.network.Network;
-import com.powsybl.iidm.network.Terminal;
-import com.powsybl.iidm.network.TwoWindingsTransformer;
+import com.powsybl.iidm.network.*;
```
[the remark this answers is not in the document]
Comment (TheoPascoli) [reply gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/shift/ScalableGeneratorConnector.java:13]: Intellij auto factorize this when there is more than 4 imports of the same package

File: gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/shift/ZonalScalableProvider.java [hunk 5]
```diff
@@ -1,13 +1,8 @@
-/*
```
[the remark this answers is not in the document]
Comment (TheoPascoli) [reply gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/shift/ZonalScalableProvider.java:1]: ok

File: gridcapa-swe-commons/src/test/java/com/farao_community/farao/gridcapa_swe_commons/shift/CountryBalanceComputationTest.java [hunk 6]
```diff
@@ -4,20 +4,20 @@
...
 import java.util.Map;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 
 /**
  * @author Ameni Walha {@literal <ameni.walha at rte-france.com>}
  */
-@SpringBootTest
+@SpringBootTest(classes = CountryBalanceComputation.class)
```
Comment (vbochetRTE) [inline gridcapa-swe-commons/src/test/java/com/farao_community/farao/gridcapa_swe_commons/shift/CountryBalanceComputationTest.java:20]: You don't need @SpringBootApplication annotation here as you test class' methods and not instance's methods

**Review by vbochetRTE — APPROVED** [review]

**Pull request merged**